sympy/integrals/quadrature.py

```diff
@@ -486,14 +509,89 @@ def gauss_jacobi(n, alpha, beta, n_digits):
     pd = p.diff(x)
     pn = jacobi_poly(n+1, alpha, beta, x, polys=True)
     xi = []
-    w  = []
+    w = []
     for r in p.real_roots():
         if isinstance(r, RootOf):
             r = r.eval_rational(S(1)/10**(n_digits+2))
         xi.append(r.n(n_digits))
         w.append((
-            - (2*n+alpha+beta+2) / (n+alpha+beta+S.One)
-            * (gamma(n+alpha+1)*gamma(n+beta+1)) / (gamma(n+alpha+beta+S.One)*gamma(n+2))
-            * 2**(alpha+beta) / (pd.subs(x, r) * pn.subs(x, r))
-        ).n(n_digits))
+            - (2*n+alpha+beta+2) / (n+alpha+beta+S.One) *
+            (gamma(n+alpha+1)*gamma(n+beta+1)) /
+            (gamma(n+alpha+beta+S.One)*gamma(n+2)) *
+            2**(alpha+beta) / (pd.subs(x, r) * pn.subs(x, r))).n(n_digits))
+    return xi, w
+
+
+def gauss_lobatto(n, n_digits):
+    r"""
+    Computes the Gauss-Lobatto quadrature [1]_ points and weights.
+
+    The Gauss-Lobatto quadrature approximates the integral:
+
+    .. math::
+        \int_{-1}^1 f(x)\,dx \approx \sum_{i=1}^n w_i f(x_i)
+
+    The nodes `x_i` of an order `n` quadrature rule are the roots of `P'_(n-1)`
+    and the weights `w_i` are given by:
+
+    .. math::
+        &w_i = \frac{2}{n(n-1) \left[P_{n-1}(x_i)\right]^2},\quad x\neq\pm 1\\
+        &w_i = \frac{2}{n(n-1)},\quad x=\pm 1
+
+    Parameters
+    ==========
+
+    n : the order of quadrature
+
+    n_digits : number of significant digits of the points and weights to return
+
+    Returns
+    =======
+
+    (x, w) : the ``x`` and ``w`` are lists of points and weights as Floats.
+             The points `x_i` and weights `w_i` are returned as ``(x, w)``
+             tuple of lists.
+
+    Examples
+    ========
+
+    >>> from sympy.integrals.quadrature import gauss_lobatto
+    >>> x, w = gauss_lobatto(3, 5)
+    >>> x
+    [-1, 0, 1]
+    >>> w
+    [0.33333, 1.3333, 0.33333]
+    >>> x, w = gauss_lobatto(4, 5)
+    >>> x
+    [-1, -0.44721, 0.44721, 1]
+    >>> w
+    [0.16667, 0.83333, 0.83333, 0.16667]
+
+    See Also
+    ========
+
+    gauss_legendre,gauss_laguerre, gauss_gen_laguerre, gauss_hermite,\
```

Comment:
I wonder if the `\` at the end here is causing the error: https://github.com/sympy/sympy/pull/11728#issuecomment-256770695